Samples/MaterialMvvmSample/Views/MaterialMenuButtonView.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<views:BaseMainView
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:views="clr-namespace:MaterialMvvmSample.Views"
+    xmlns:material="clr-namespace:XF.Material.Forms.UI;assembly=XF.Material"
+    x:Class="MaterialMvvmSample.Views.MaterialMenuButtonView"
+    >
+
+    <ContentPage.Content>
+        <StackLayout>
+            <material:MaterialMenuButton ButtonType="Outlined" CornerRadius="24" Choices="{Binding Actions}" Margin="10" />
```

Comment:
You are binding to Actions, but where is Actions?

Reply:
In the MainViewModel